benchmarking/daint_single_node/stdout_to_json.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+
+# This script parses the standard output of a FV3GFS run to extract timing
+# information and convert it to the JSON format which is used by the performance
+# monitoring script.
+
+# 2021/01/22 Oliver Fuhrer, Vulcan Inc, oliverf@vulcan.com
+
+import os, re, click, glob, time, datetime, json, sys
+
+# this dict maps the names of the timer in the JSON file to the actual
+# timers reported in stdout of FV3GFS, multiple entries are accumulated
+TIMER_MAPPING = {
+  "total": ["Total runtime"],
+  "init": ["1-Initialization", "2-Main-loop-1st-trip"],
+  "main_loop": ["3-Main-loop"],
```

Comment:
```suggestion
  "main": ["3-Main-loop"],
```

Reply:
I actually disagree on this one. Could we retain `main_loop` as specified in https://paper.dropbox.com/doc/Performance-timers-Fortran-Python--BD4~XI4SNtfc~gBml74rb_XUAg-lSnQfJGiZ6LVfSLQpcnwT ?

Reply:
if we agree on that we need a parallel PR in fv3core to this. otherwise we break things